pkg/asset/internal/templates.go

```diff
@@ -539,12 +540,16 @@ spec:
     spec:
       containers:
       - name: etcd-operator
-        image: quay.io/coreos/etcd-operator:c391d8b7638deb81aa877773a0acce389f602415
+        image: quay.io/coreos/etcd-operator:dev
```

Comment:
we will change this to release 0.2.1 tomorrow. we just want to make sure current etcd operator works with bootkube as it is now. then we bump it, then we change it here. it is kind of chicken egg problem for now.